relay-config/src/config.rs

```diff
@@ -769,6 +784,11 @@ struct Cache {
     file_interval: u32,
     /// Interval for evicting outdated project configs from memory.
     eviction_interval: u32,
+    /// The settings to configure persistent buffering.
+    ///
+    /// When enabled all the incoming envelope will be persisted to the disk instead of keeping
+    /// them in memory.
```

Comment:
```suggestion
    /// When enabled all incoming envelopes will be persisted to disk instead of keeping
    /// them in memory.
```

I'd even go one step further and not comment about implementation details. What this will do is persist to disk, but it'll be possible to have some stuff in memory.

```suggestion
    /// When enabled all incoming envelopes will be persisted to disk.
```